src/features/user/RegisterTrees/RegisterTreesWidget.tsx

```diff
@@ -232,10 +232,10 @@ function RegisterTreesForm({
   }
 
   React.useEffect(() => {
-    if (contextLoaded) {
+    if (contextLoaded && user?.type === 'tpo') {
       loadProjects();
     }
-  }, [contextLoaded]);
+  }, [contextLoaded, user]);
```

Comment:
@sunilsabatp the API is called twice at times. Probably because of the dependencies.